charts/ceph-csi-cephfs/templates/nodeplugin-daemonset.yaml

```diff
@@ -59,24 +59,23 @@ spec:
           resources:
 {{ toYaml .Values.nodeplugin.registrar.resources | indent 12 }}
         - name: csi-cephfsplugin
-          securityContext:
-            privileged: true
-            capabilities:
-              add: ["SYS_ADMIN"]
-            allowPrivilegeEscalation: true
           image: "{{ .Values.nodeplugin.plugin.image.repository }}:{{ .Values.nodeplugin.plugin.image.tag }}"
-          args :
+          imagePullPolicy: {{ .Values.nodeplugin.plugin.image.pullPolicy }}
+          args:
             - "--nodeid=$(NODE_ID)"
             - "--type=cephfs"
+{{- if or (eq .Values.nodeplugin.plugin.image.tag "canary") (semverCompare ">=1.2.0" .Values.nodeplugin.plugin.image.tag | default 0) }}
```

Comment:
remove if the check for this one as we release helm templates for each version

Reply:
Wouldn't it be nice to support v1.1.0 images with this new charts?

Reply:
it recommended using helm chats from the released version. not to use 1.2.0 helm chats for 1.1.0 deployments

Reply:
Nah I get the point, it would introduce a spaghetti of if statements, will remove :)